modules/tinymce/src/themes/silver/main/ts/api/Options.ts

```diff
@@ -257,6 +257,11 @@ const register = (editor: Editor): void => {
     default: true
   });
 
+  registerOption('promotion', {
```

Comment:
This new option needs adding in the `OptionTypes.ts` module so it's in the bundled types.